src/main/scala/xebia/cakemix/AskTimeoutProvider.scala

```diff
@@ -17,7 +17,9 @@ trait AskTimeoutProvider {
 }
 
 object AskTimeoutProvider {
-  def fromConfig(config: Config, key: String): Timeout = {
+  private val AskTimeoutConfigKey = "xebia.cakemix.ask-timeout"
```

Comment:
In the scala world it is getting less and less common to use fully specified package names with com/org/etc. Akka has `akka._`, Spray has `spray._`, Scalatest has `scalatest`, etc.